mantidproject/mantid
# Create IDF for new GPSANS name (CG2)

The GPSANS instrument at HFIR is changing its name to CG2.
We are adding a new instrument definition, identical to the HiResSANS_Definition.xml

Since this is a simple change I didn't create a ticket, all the info needed is in this PR.

To test: 
- Load a new GPSANS file obtained from the instrument scientist or DAS group.
- Try to reduce the data from the reduction interface. It should work using the "GPSANS" reduction UI.

No release notes needed: mentioning this should be part of the broader changes to GPSANS.
